documentation/docs/developer_guide/documentation_guidelines/index.md

```diff
@@ -1,18 +1,37 @@
 # Documentation guidelines
 
+First of all, you should become a member of the CABLE-LSM GitHub organisation prior to contribute to the CABLE's documentation. Please ask for membership [on this issue][cable-lsm-join]. All guidelines are written with the assumption you are a member of the organisation.
+
 The documentation for CABLE has two parts:
 
 - inline documentation in the source code. This documentation includes the detailed scientific documentation. It is written using [FORD][ford].
 - the standalone guides such as the User Guide and the Developer Guide. It is written using [Material for MkDocs][material].
 
-All documentation is located in the [CABLE GitHub repository][cable-repo]. Both tools use [Markdown][md-cheatsheets] and [LaTeX Mathematics][latex-maths] for formatting. You can find a cheatsheet for FORD [on this page][ford-cheatsheets].
+Both tools use [Markdown][cheat-sheets] and [LaTeX Mathematics][latex-maths] for formatting. 
+
+## Documentation workflow
+All documentation is located in the [CABLE GitHub repository][cable-repo]. To update the documentation, you need to follow a traditional GitHub contribution process: open an issue and a new branch; work on your changes; submit a pull request for review before publication. The process is explained in more details [on this page][git-process].
+
+## Scientific documentation
+The *scientific documentation* should be added directly into the source code available under the `src/` directory. Please use [these guidelines][api-guidelines] to structure the documentation. You can find a cheatsheet for FORD [on this page][cheat-sheets].
```

Comment:
Move scientific description section of user guide to the top level.

Also what will be in the "Scientific description" section? General description like PFTs? Model layers? Should we say somewhere something like: documentation that does not belong in the source code/API would go there?

Reply:
What will be in the scientific description section of the user guide is still to be determined. Probably once we have written the detailed documentation in the source code, it would be good to have a short overview of the science covered by CABLE. But that's still open to debate.

Reply:
I think I will leave the Scientific description section where it is for now. Let's see what goes in there first and then we'll decide where to put it.